### ISSUE TITLE
Electricity production from waste incinerator is included in must-run CHP entry in merit order table

### ISSUE THREAD

**Issue by jorisberkhout** [quintel/etmodel#2104]
I noticed that the electricity produced by the waste incinerator (not the waste CHP) is included in the must-run CHP entry in the merit order table. There seem to be two issues with this waste incinerator:
- it takes part in the merit order calculation as

```
- merit_order.type = must_run
- merit_order.group = industry_chp
```
- it is part of the `must_run_electricity_production` group, which is why it is displayed under the must-run entry in the merit order table.

Long story short, should the waste incinerator be must-run, @AlexanderWirtz or @ChaelKruip ? Or is it dispatchable?

**Comment by AlexanderWirtz** [comment]
It should be dispatchable, but with baseload marginal costs. :-)

**Comment by AlexanderWirtz** [comment]
We can ask Albert Bloem how this works?

**Comment by TessColijn** [comment]
@AlexanderWirtz  has Albert Bloem been asked? Or is there any update on wether waste incinerators are must-runs or dispatchables?

**Comment by AlexanderWirtz** [comment]
Funny, I think @jorisberkhout was the last one to talk to Albert Bloem about this.

**Comment by michieldenhaan** [comment]
Closing, fixed with recent heat update
